src/modules/shared/pages/innovation/actions/task-details.component.html

```diff
@@ -0,0 +1,139 @@
+                  ></path>
+                </svg>
+              </a>
+            </li>
+            <li class="nhsuk-pagination-item--next" *ngIf="taskNumber < tasksIds.length - 1">
+              <a class="nhsuk-pagination__link nhsuk-pagination__link--next cursor-pointer" (click)="handlePagination('next')">
+                <span class="nhsuk-pagination__title">Next</span>
+                <span class="nhsuk-u-visually-hidden">:</span>
+                <span class="nhsuk-pagination__page">Requested task</span>
+                <svg class="nhsuk-icon nhsuk-icon__arrow-right" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true" width="34" height="34">
+                  <path 
+                    d="M19.6 11.66l-2.73-3A.51.51 0 0 0 16 9v2H5a1 1 0 0 0 0 2h11v2a.5.5 0 0 0 .32.46.39.39 0 0 0 .18 0 .52.52 0 0 0 .37-.16l2.73-3a.5.5 0 0 0 0-.64z">
+                  </path>
+                </svg>
+              </a>
+            </li>
+          </ul>
+        </nav>
+      </ng-container>
+    </div>
+
+    <div class="nhsuk-grid-column-one-third" *ngIf="!isAdmin">
+      <div class="nhsuk-card x-card-top-border">
+        <div class="nhsuk-card__content">
+          <h2 class="nhsuk-card__heading nhsuk-heading-m">What you can do</h2>
+          <ng-container *ngIf="isInnovatorType">
+            <ul class="nhsuk-list">
+              <li>
+                <a *ngIf="task.status === 'OPEN'" routerLink="/innovator/innovations/{{ innovationId }}/tasks/{{ task.id }}/accept" class="nhsuk-button nhsuk-u-margin-right-3 nhsuk-u-margin-bottom-3">
+                  Mark task as done
+                </a>
+                <a *ngIf="task.status === 'OPEN'" routerLink="/innovator/innovations/{{ innovationId }}/tasks/{{ task.id }}/decline" class="nhsuk-button nhsuk-button--secondary nhsuk-u-margin-bottom-3">
+                  Decline action
+                </a>
+              </li>
+              <li class="nhsuk-u-margin-bottom-4">
+                <a routerLink="/innovator/innovations/{{ innovationId }}/threads"> View messages about this task and send a reply</a>
+              </li>
+              <li>
+                <a *ngIf="task.status === 'OPEN'" routerLink="/innovator/innovations/{{ innovationId }}/record/sections/{{ task.section }}">Go to section to complete this task</a>
+                <a *ngIf="task.status !== 'OPEN'" routerLink="/innovator/innovations/{{ innovationId }}/record/sections/{{ task.section }}">Go to section to review</a>
+              </li>
+            </ul>
+          </ng-container>
+
+          <ng-container *ngIf="isAccessorType || isAssessmentType">
+            <ul class="nhsuk-list">
+              <li>
+                <a *ngIf="task.status !== 'OPEN' && task.sameOrganisation" routerLink="/{{ userUrlBase }}/innovations/{{ innovationId }}/tasks/{{ task.id }}/reopen" class="nhsuk-button nhsuk-u-margin-right-3 nhsuk-u-margin-bottom-3">
```

Comment:
I think the condition here is missing the check for `CANCELLED` tasks since they can't be reopened as-well.